source/source/deengames/planetoid/component/MouseShootComponent.hx

```diff
@@ -0,0 +1,26 @@
+package deengames.planetoid.component;
+
+import flixel.FlxSprite;
+import flixel.FlxG;
+
+import nebula.ecs.component.AbstractComponent;
+import flixel.addons.weapon.FlxWeapon;
+import flixel.addons.weapon.FlxBullet;
+import flixel.util.helpers.FlxBounds;
+
+import flixel.math.FlxPoint;
+
+class MouseShootComponent extends AbstractComponent
```

Comment:
I agree that this is a better name, but I am still not sure how is this `nebula`